vsix/ItemTemplates/BlankPage/BlankPage.h

```diff
@@ -1,17 +1,11 @@
 ﻿#pragma once
-
```

Comment:
This blank line established a stylistic convention in existing C++/WinRT codebases, it should be kept.

Reply:
However that isn't consistent with the generated files from IDL file which doesn't have a space after #pragma once

Reply:
Fair enough, I haven't noticed